std/file.d

```diff
@@ -4970,24 +4957,45 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
 {
     string[] listdir(string pathname)
     {
-        import std.algorithm;
-        import std.array;
-        import std.file;
-        import std.path;
+        import std.algorithm.iteration : filter, map;
+        import std.array : array;
+        import std.path : baseName;
 
-        return std.file.dirEntries(pathname, SpanMode.shallow)
+        return dirEntries(pathname, SpanMode.shallow)
             .filter!(a => a.isFile)
-            .map!(a => std.path.baseName(a.name))
+            .map!(a => baseName(a.name))
             .array;
     }
 
-    void main(string[] args)
+    void main()
     {
         import std.stdio;
 
-        string[] files = listdir(args[1]);
-        writefln("%s", files);
-     }
+        string[] files = listdir(".");
+        writeln(files);
+    }
+}
+
+/// Compile D files in parallel:
+version (StdDdoc)
+@system unittest
+{
```

Comment:
I think, this unittest introduced [issue 21796](https://issues.dlang.org/show_bug.cgi?id=21796).

See [logfile of circle-ci](https://app.circleci.com/pipelines/github/dlang/phobos/3976/workflows/90ccdf2d-ff37-4e70-b7da-ca80d626c604/jobs/13524).